### ISSUE TITLE
Generate arbitrary length

### ISSUE THREAD

**Issue by jh27kim** [kwsong0113/diffusion-forcing-transformer#15]
Hi @kwsong0113 

Thank you for releasing the codebase for your paper. It's very impressive work.

I am trying to generate an arbitrary-length video and attempted to do so by setting dataset.n_frames to a value greater than 200. However, this results in a `num_samples=0` error when loading the dataloader. I suspect I'm missing a config in the process. 

Could you please share the correct command-line configuration to generate long videos, as demonstrated in your paper?

The command i used is the one specified in the main repo:
```python -m main +name=single_image_to_long dataset=realestate10k_mini algorithm=dfot_video_pose experiment=video_generation @diffusion/continuous load=pretrained:DFoT_RE10K.ckpt 'experiment.tasks=[validation]' experiment.validation.data.shuffle=True dataset.context_length=1 dataset.frame_skip=1 dataset.n_frames=300 algorithm.tasks.prediction.keyframe_density=0.0625 algorithm.tasks.interpolation.max_batch_size=4 experiment.validation.batch_size=1 algorithm.tasks.prediction.history_guidance.name=stabilized_vanilla +algorithm.tasks.prediction.history_guidance.guidance_scale=4.0 +algorithm.tasks.prediction.history_guidance.stabilization_level=0.02  algorithm.tasks.interpolation.history_guidance.name=vanilla +algorithm.tasks.interpolation.history_guidance.guidance_scale=1.5```

Thanks!

**Comment by kwsong0113** [comment]
It seems like you're trying to do image-to-video inference. Could you try appending `dataset.filter_min_len=1` to the command? [By default, `filter_min_len` is set to `null`, which filters out all videos shorter than 300 frames--this is why `n_samples=0`](https://github.com/kwsong0113/diffusion-forcing-transformer/blob/566e7d2dd81930a59885fab45dc65daeb48f82c5/configurations/dataset/base_video.yaml#L28). Since you only need the first frame, setting `filter_min_len` to 1 should resolve the issue.

**Comment by jh27kim** [comment]
Thank you, @kwsong0113 for the quick response.

For others who encounter the same issue, here is a temporary fix.

For pose-conditioned generation, the maximum number of frames in RealEstate10K_mini dataset is less than 300 (279 frames). So, set dataset.n_frames=279 and algorithm.tasks.prediction.keyframe_density=0.0575 to avoid exception.

For pose-free generation (using `algorithm=dfot_video`), it appears the provided checkpoint has a different architecture. I looked through the wiki but could not find the checkpoint. Is there any other path for the checkpoint @kwsong0113 ? Thank you!

**Comment by kwsong0113** [comment]
@jh27kim We do not have a checkpoint trained exclusively for pose-free generation on the RealEstate10K dataset. [The Re10K model can generate without pose because it is trained with random dropout of pose conditioning](https://github.com/kwsong0113/diffusion-forcing-transformer/blob/566e7d2dd81930a59885fab45dc65daeb48f82c5/algorithms/dfot/backbones/u_vit/u_vit3d_pose.py#L62-L68), but I don't think the generated videos will have good quality in this case due to insufficient conditioning.

**Comment by jh27kim** [comment]
Right that makes sense. Thank for clarifying.
